File: server/models/retailer.js
module.exports = (sequelize, Sequelize) => {
    const Retailer = sequelize.define("retailer", {
        name: {
            type: Sequelize.STRING,
            required: true,
        }, 
        contact: {
            type: Sequelize.INTEGER
        },
        address: {
            type: Sequelize.STRING,
            required: true,
            unique: true
        },
        location: {
            type: Sequelize.STRING,
            required: true,
            unique: true
        },
    }, {
        sequelize,
        freezeTableName: true,
        tableName: 'retailer',
        timestamps: false,
        underscored: true
    },
    )
    return Retailer;
}

File: lib/models/product/retailer.dart
import 'package:pos/models/product/retailer_bill.dart';

class Retailer {
  Retailer({
    this.name,
    this.id,
    this.contact,
    this.address,
    this.location,
    this.retailerBill,
  });
  String? name;
  String? id; // Auto
  String? contact;
  String? address;
  String? location;
  List<RetailerBill>? retailerBill;
}
